engine/src/test/java/com/redhat/cloud/notifications/routers/ReplayResourceTest.java

```diff
@@ -85,6 +85,7 @@ void beforeEach() {
         // OrgId: 123
 
         when(endpointRepository.getTargetEndpoints(eq(ORG_ID_1), eq(eventType))).thenReturn(List.of(endpoint));
+        when(endpointRepository.getTargetEndpointsWithoutUsingBgs(eq(ORG_ID_1), eq(eventType))).thenReturn(List.of(endpoint));
```

Comment:
Is this needed? Everything is disabled by default in the configuration, so getTargetEndpointsWithoutUsingBgs should not be called by this test.